spec/helpers/service_helper/textual_summary_spec.rb

```diff
@@ -0,0 +1,30 @@
+describe ServiceHelper::TextualSummary do
+  context ".textual_orchestration_stack" do
+    let(:os_cloud) { FactoryGirl.create(:orchestration_stack_cloud, :name => "cloudstack1") }
+    let(:os_infra) { FactoryGirl.create(:orchestration_stack_openstack_infra, :name => "infrastack1") }
+
+    before do
+      login_as @user = FactoryGirl.create(:user)
+    end
+
+    subject { textual_orchestration_stack }
+    it 'contains the link to the associated stack' do
```

Comment:
Why are you using the same `it` twice in the same context? Please create a nested-context for both cases.